app-shells/mksh/mksh-9999.ebuild

```diff
@@ -12,29 +12,31 @@ if [[ $PV = 9999 ]]; then
 	ECVS_USER="_anoncvs"
 	ECVS_AUTH="ext"
 	KEYWORDS=""
-	DEPEND="static? ( dev-libs/klibc )"
 else
-	inherit unpacker
-	DEPEND="app-arch/cpio
-		static? ( dev-libs/klibc )"
-	SRC_URI="http://www.mirbsd.org/MirOS/dist/mir/mksh/${PN}-R${PV}.cpio.gz"
-	KEYWORDS="~amd64 ~ppc ~x86 ~amd64-linux"
+	SRC_URI="http://www.mirbsd.org/MirOS/dist/mir/mksh/${PN}-R${PV}.tgz"
+	KEYWORDS="~amd64 ~ppc ~x86 ~amd64-linux ~x86-linux"
 fi
 
 DESCRIPTION="MirBSD Korn Shell"
 HOMEPAGE="http://mirbsd.de/mksh"
 LICENSE="BSD"
 SLOT="0"
 IUSE="static"
+DEPEND="static? ( dev-libs/klibc )"
 RDEPEND=""
 S="${WORKDIR}/${PN}"
 
 src_compile() {
 	tc-export CC
 	# we want to build static with klibc
-	if use static; then unset CC; export CC="/usr/bin/klcc"; export LDSTATIC="-static"; fi
+	if use static
+	then
+		unset CC
+		export CC="/usr/bin/klcc"
+		export LDSTATIC="-static"
+	fi
 	export CPPFLAGS="${CPPFLAGS} -DMKSH_DEFAULT_PROFILEDIR=\\\"${EPREFIX}/etc\\\""
-	sh Build.sh -r -c lto || sh Rebuild.sh || die
+	sh Build.sh -r -c lto || die
 }
 
 src_install() {
```

Comment:
I think the canonical way would be:

    into /
    dobin mksh

Other helpers shouldn't be affected by the prefix.